tools/nl/embeddings/utils.py

```diff
@@ -267,11 +277,13 @@ def get_or_download_file_from_gcs(ctx: Context, file_name: str) -> str:
   If the file is already downloaded, it returns the file path.
   Otherwise, it downloads the file from GCS to the local file system and returns that path.
   """
+  bucket_name = DEFAULT_MODELS_BUCKET
```

Comment:
Can this function use the one that nl server also uses?

Reply:
Updated to using the shared lib now. However it needs to be imported awkwardly. PTAL.